mon-pix/app/components/chips-choice.js

```diff
@@ -0,0 +1,9 @@
+import Component from '@glimmer/component';
+
+export default class ChipsChoice extends Component {
```

Comment:
Je propose `ChoiceChip` : 
- nom de composant au singulier
- plus correct grammaticalement IMO

```suggestion
export default class ChoiceChip extends Component {
```

Reply:
C'est fait :D

Reply:
Top 👌